api/opencl_platform_layer.asciidoc

```diff
@@ -731,12 +731,21 @@ Also see extension *cl_khr_fp64*.
         double precision floating-point capability is 0.
 
         If double precision is supported by the device, then the minimum
-        double precision floating-point capability is:
+        double precision floating-point capability is (from OpenCL 2.0 onwards):
```

Comment:
Nit-picky comment: Most places in the spec talk about "OpenCL X.Y devices" rather than "OpenCL X.Y" generally.  Consider instead:

> ... then the minimum double precision floating-point capability for OpenCL 2.0 or newer devices is: